packages/data/lib/core/api_key.dart

```diff
@@ -0,0 +1,14 @@
+import 'dart:convert';
+
+abstract class ApiHelperCore {
```

Comment:
Abstract class cannot be instantiated.

Reply:
why? i am implementing object in abstract class I am just using to access to my const values